Sources/TuistDependenciesTesting/Carthage/Models/CarthageVersionFile+TestData.swift

```diff
@@ -577,12 +595,41 @@ extension CarthageVersionFile {
         }
         """
     }
+
+    static var testAhoyRTCJson: String {
+        """
+        {
+          "Mac" : [
+
+          ],
+          "watchOS" : [
+
+          ],
+          "tvOS" : [
+
+          ],
```

Comment:
`Carthage` returns empty arrays for unwanted platforms? 🤔

Reply:
For this dependency it does, I'm not sure why, but that's a real snapshot

Reply:
that is fine then :)